blocks/snippets.py

```diff
@@ -0,0 +1,14 @@
+"""Routines to build frequently used Theano expressions."""
+from theano import tensor
+
+
+def L2_norm(tensors):
+    """Computes the total L2 norm of a set of tensors.
+
+    Parameters
+    ----------
+    tensors : iterable of :class:`~theano.Variable`
+        The tensors.
+
+    """
+    return tensor.sqrt(sum((t ** 2).sum() for t in tensors))
```

Comment:
> theano_expressions then? The thing is that a "snippet" is almost always used to refer to a piece of code that you copy 
> and paste, which is the opposite of what you're doing here.

Literately it is, but this file is exactly to avoid snippets, so this is how that name is carried over to the file. 

>  which takes multiple inputs, like all element-wise operators in Theano.
Okay, but why does it have a single output then, which it apparently has according to the snippet you propose.

> I'd also be surprised if the documentation of tensor.sqr is longer than 10 words

Try `tensor.sqr?`, make some tea of coffee, and enjoy reading. I am not saying that theano is badly documented though.

Smaller CG is the decisive argument for me, will change this one.